flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/pipeline/Predictor.scala

```diff
@@ -121,7 +121,7 @@ object Predictor {
 
         input.mapWithBcVariable(model){
           (element, model) => {
-            (element, predictOperation.predict(element, model))
+            (element, predictOperation.predict(element, model, resultingParameters))
```

Comment:
In this case the output of `predict` depends on the parameter settings. With access only to `value` and `model` we can't know what to output otherwise.